data/json/items/gun/nail.json

```diff
@@ -1,5 +1,29 @@
 [
   {
+    "id": "coilgun",
+    "copy-from": "gun_base",
+    "type": "GUN",
+    "name": "coilgun",
+    "description": "A homemade gun, using electromagnets to accelerate a ferromagnetic projectile to high velocity.  Powered by UPS.",
+    "weight": 3341,
+    "volume": 11,
+    "price": 75000, "//": "Hard to make, plentiful and cheap ammo, and silent - people will want this thing.",
+    "bashing": 10,
+    "to_hit": -2,
+    "material": "steel",
+    "skill": "rifle",
+    "ammo": "nail",
+    "ranged_damage": 8,
```

Comment:
Crafting difficulty isn't a simple sliding scale. Ability to craft a
working coilgun doesn't imply ability to make factory-quality sights for
that same gun.

Maybe they can, but that is represented by the ability to craft and install
a sight mod, not by boosting the sight quality of every weapon based on how
difficult they are to make.

Generally, you can't assume that someone can craft something because they
are capable of crafting something "more difficult".